BUILDING.md

```diff
@@ -37,15 +33,73 @@ Building: Docker >> alpine-linux-x64 >> NODE_ENABLED
 Override build-steps ? (leave empty to run pre-configured steps): j2v8
```

Comment:
Good catch, I'll rather put in `nodejs j2v8 test` as an example, as it better illustrates that one can specify multiple steps here.